apiserver/facades/client/modelmanager/modelmanager_test.go

```diff
@@ -205,14 +205,21 @@ func (s *modelManagerSuite) TestCreateModelArgs(c *gc.C) {
 		"ReloadSpaces",
 		"GetBackend",
 		"Model",
-		"LastModelConnection",
-		"LastModelConnection",
-		"LastModelConnection",
 		"AllMachines",
 		"LatestMigration",
 		"Close",
 	)
 
+	// Check that Model.LastModelConnection is called three times
+	// without making the test depend on other calls to Model
+	n := 0
+	for _, call := range s.st.model.Calls() {
+		if call.FuncName == "LastModelConnection" {
+			n = n + 1
```

Comment:
Best to use atomic.AddInt

Reply:
I don't see why that's necessary here, there shouldn't be any concurrent access. Can you explain what I'm missing?

Reply:
We prefer to see that in tests to ensure we don't have 'go test -race' issues :D

Reply:
This really shouldn't have a race, there are no separate goroutines incrementing the variable. I see the other places in the suite where it's used, and they all involve counters that are updated in separate goroutines or in concurrent network handlers.
I think we're OK here, but if there's something specific I'm missing, I'm happy to learn.